### PR TITLE
Cmd +e / Ctrl + e shows empty slide

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Earlier clicking on escape closed settings that code was commented. However for issue #245 we want to show an empty slide when clicked on escape button.

### CODE REVIEW

File: www/js/index.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ function escKey() {
   /* if (settings.$settings.classList.contains('animated')) {
     settings.closeSettings();
   } */
+  // show Empty Slide
+  global.controller.sendText('');
```
Comment (navdeepsinghkhalsa) [inline www/js/index.js]: This is being bound with bindGlobal, which will tigger this even if `esc` is pressed while the cursor is in a text field. That doesn't seem intuitive. @maneetpaul @saintsoldierx Do you have any thoughts on what criteria need to be met for the `esc` key to trigger an empty slide?

Comment (inderpreetsingh) [reply www/js/index.js]: I agree! Its bad UX to repurpose keystrokes like that. I think we should do something like `cmd + e` or `ctrl+e` for this.

**Comment by saintsoldierx** [comment]
Having discussed with Inder Singh, Esc key can present other issues when using STTM - so let's use CMD + E for now as hotkey. Also, this should toggle the secondary screen on and off - not just show a blank screen.

**Comment by maneetpaul** [comment]
@saintsoldierx can you clarify what you mean by “turn off the second display”? My original intention of this ticket was a quick way to clear all the text off the slide (while retaining the background of the theme). I don’t want to change the scope to actually turn off the entire display.

**Comment by inderpreetsingh** [comment]
@saintsoldierx I agree with maneetpaul veerji we should have a separate functionality for closing secondary window. we talked about having a close icon on viewer preview, clicking on this icon would dim the viewer preview (showing it as its disabled) and close the secondary viewer window.

**Comment by inderpreetsingh** [comment]
Changed from esc to cmd+e

**Comment by ManjotS** [comment]
what is windows equivalent?

**Comment by inderpreetsingh** [comment]
@ManjotS it's `ctrl + e`

File: www/js/index.js (hunk 1 above)
Comment (navdeepsinghkhalsa) [reply www/js/index.js]: This was later decided to be cmd/ctrl+e, so I will click "resolve" for this conversation

**Pull request merged**